includes/ApplHandlingByStr.php

```diff
@@ -93,6 +172,22 @@ function str_isAccepted ( $stat_str, $user )
 }
 
 
+function str_isForwarded( $stat_str, $user )
+{
+	$STR = explode ( ';', $stat_str );
+	for ( $i = 1 ;  $i < count ( $STR ) ; $i++ )
+	{
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->